lr_face/data_providers.py

```diff
@@ -136,11 +161,13 @@ def make_pairs(X: np.ndarray, y: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
                 # make same-person pairs by pairing all images of the person
                 pairs.append((imgs[i], imgs[j]))
                 same_different_source.append(1)
+                ids.append(hash((imgs[i], imgs[j])))
```

Comment:
hash(np.concatenate((imgs[i], imgs[j])).tostring())

Reply:
is weg